src/WebManager/app/src/components/repository.tsx

```diff
@@ -1,20 +1,20 @@
 ﻿import * as React from "react";
-import { browserHistory } from "react-router";
+import history from "./history";
 import { Breadcrumb, IBreadcrumbItem } from "office-ui-fabric-react/lib/Breadcrumb";
 
 import { Docker } from "../services/docker";
 import { AuthBanner } from "./auth-banner";
 import { RepositoryTagViewer } from "./repository-tag-viewer";
 
-export interface IRepositoryProps { params: any }
+export interface IRepositoryProps { match: any }
```

Comment:
Because react used to have the params file so in case you wanted to access the properties you would do it as this.props.params, but it changed to match so I did this change, if not the code would not compile